keras_cv/models/segmentation/__internal__/segmentation_head.py

```diff
@@ -61,15 +61,30 @@ def __init__(
         filters=256,
         activations="relu",
         output_scale_factor=None,
+        dropout=0.0,
+        kernel_size=3,
+        use_bias=False,
         **kwargs,
     ):
-        """"""
+        """
+        Args:
+            classes: the number of possible classes for the segmentation map
+            convs: default 2; the number of conv blocks to use in the head (conv2d-batch_norm-activation blocks)
+            filters: default 256; the number of filters in each Conv2D layer
+            activations: default 'relu'; the activation to apply in conv blocks
+            output_scale_factor: default None; the scale to apply in the UpSampling call before the output
+            dropout: default 0.0; the dropout to apply between each conv block
+            **kwargs:
```

Comment:
add docstring for `kernel_size` and `use_bias`?

Reply:
Done